### ISSUE TITLE
Issue with table where a column is spanned over rows

### ISSUE THREAD

**Issue by oggolder** [FyiurAmron/sortablejs#4]
If you use a table like the following...

```
<sortable r1>
^Col1^Col3^Col3^
|1|one|25/12/2008|
|2|two|10/12/2008|
|3a|three|15/12/2009|
|3b|three|:::|
</sortable>
```

an extra column appears, and the order in the 3rd column is wrong.

**Comment by FyiurAmron** [comment]
I can confirm this, but honestly, there doesn't seem any reasonable way in which this could've been fixed; sorting doesn't make sense if there are spans, because, frankly, *how should it cope with them?* Sorting works by reordering the elements, and if we decide that the elements of a span (cells)
a) can't be reordered - then the sorting fails,
b) can be reordered - then the span is broken.

As such, it's a WONTFIX/CANTFIX for me, unless you (or anyone else) propose a sensible algo to solve this particular case.

**tl;dr** in current implementation, sorting tables with spans is https://en.wikipedia.org/wiki/Undefined_behavior , and IMO it should stay that way. Either don't put spans in sortable tables, or don't sort them if they have to have spans.

BTW, your example allowed me to find a bug in the current implementation of date sorting - so it was not all for naught, mate.
